lib/improver/tests/acceptance.py

```diff
@@ -142,7 +142,7 @@ def nccmp(output_path, kgo_path,
            str(output_path), str(kgo_root() / kgo_path)]
     print(" ".join(cmd))
     completion = subprocess.run(cmd,
-                                timeout=15,
+                                timeout=30,
```

Comment:
When you run pytest it fails for the global tests because it takes more than 15s. This is because the global data are big files. Ben felt 15s was too restrict and 30s would be OK. Hence the change.  The alternative solution is to thin out the global data to make the CLI test run faster but I don't think that is part of this PR. I could revert to 15s and let these test continue to fail but would prefer to leave this as is.

Reply:
Ok. If Ben's happy, then so am I.